src/components/PlaylistExporter.jsx

```diff
@@ -0,0 +1,88 @@
+import $ from "jquery" // TODO: Remove jQuery dependency
+import { saveAs } from "file-saver"
+
+import { apiCall } from "helpers"
+
+// Handles exporting a single playlist as a CSV file
+var PlaylistExporter = {
+  export: function(access_token, playlist) {
+    this.csvData(access_token, playlist).then(function(data) {
+      var blob = new Blob([ data ], { type: "text/csv;charset=utf-8" });
+      saveAs(blob, this.fileName(playlist), true);
+    }.bind(this))
+  },
+
+  csvData: function(access_token, playlist) {
+    var requests = [];
+    var limit = playlist.tracks.limit || 100;
+
+    for (var offset = 0; offset < playlist.tracks.total; offset = offset + limit) {
+      requests.push(
+        apiCall(playlist.tracks.href.split('?')[0] + '?offset=' + offset + '&limit=' + limit, access_token)
```

Comment:
you should use a literals template for this case:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

```ts
apiCall(`${playlist.tracks.href.split('?')[0]}?offset=${offset}&limit=${limit}`, access_token)
```